monkey/monkey_island.py

```diff
@@ -2,20 +2,25 @@
 
 gevent_monkey.patch_all()
 
-from monkey_island.cc.main import main
+from monkey_island.cc.main import main  # noqa: E402
+from monkey_island.cc.environment.environment_config import DEFAULT_SERVER_CONFIG_PATH  # noqa: E402
 
 
 def parse_cli_args():
     import argparse
-    parser = argparse.ArgumentParser(description="Infection Monkey Island CnC Server. See https://infectionmonkey.com")
+    parser = argparse.ArgumentParser(description="Infection Monkey Island CnC Server.  See https://infectionmonkey.com",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument("-s", "--setup-only", action="store_true",
                         help="Pass this flag to cause the Island to setup and exit without actually starting. "
                              "This is useful for preparing Island to boot faster later-on, so for "
                              "compiling/packaging Islands.")
+    parser.add_argument("-c", "--config", action="store",
+                        help="The path to the server configuration file.",
+                        default=DEFAULT_SERVER_CONFIG_PATH)
     args = parser.parse_args()
-    return args.setup_only
+    return (args.setup_only, args.config)
```

Comment:
Instead of passing on tuples (which is not very extendable) we should either pass back `args` or even better, create a dataclass of `IslandCmdArgs`. That way is more extendable and we can type hint instead of having to mentally map that `parse_cli_args()[0]` means if island is setup only and `parse_cli_args()` is config path

Reply:
Can we address this in a later PR? There are further changes to this code and we can make a more informed decision about how to proceed.

Reply:
Sure

Reply:
Added eabcb3219734bf9415a34d0070422c7d6b5d71c2 so we don't forget.